spec/GraphQLOverHTTP.md

```diff
@@ -267,25 +274,26 @@ still leveraging modern features if available in the server.
 
 ## GET
 
-For HTTP GET requests, the GraphQL request parameters MUST be provided in the
-query component of the request URL, encoded in the
+For HTTP GET requests, the _GraphQL-over-HTTP request_ parameters MUST be
+provided in the query component of the request URL, encoded in the
 `application/x-www-form-urlencoded` format as specified by the
 [WhatWG URLSearchParams class](https://url.spec.whatwg.org/#interface-urlsearchparams).
 
-The `query` parameter MUST be the string representation of the Source Text of
-the Document as specified in
+The {query} parameter MUST be the URL-encoded string representation of the
```

Comment:
Nope, you (and Past Benjie) are right. I’m not 100% certain we’re defining things in the right way; we might need a “decoding” step to be explicit, because there’s ambiguity around e.g. `variables` the GraphQL-over-HTTP request parameter (map) and `variables` the query parameter (JSON encoded string). I might need to introduce a different clearer term.

Reply:
This has been addressed via a note.